drivers/ovsconstants.go

```diff
@@ -0,0 +1,43 @@
+package drivers
+
+const (
+	operCreateBridge oper = iota
+	operDeleteBridge
+	operCreatePort
+	operDeletePort
+)
+
+const (
```

Comment:
I like the iota’s to be in a separate block so it’s obvious they all belong to the same group, otherwise I have no preferences.

-Erik

> On May 26, 2015, at 10:26 AM, Madhav Puri <notifications@github.com> wrote:
> 
> In drivers/ovsconstants.go <https://github.com/contiv/netplugin/pull/82#discussion_r31057914>:
> 
> > @@ -0,0 +1,43 @@
> > +package drivers
> > +
> > +const (
> > +	operCreateBridge oper = iota
> > +	operDeleteBridge
> > +	operCreatePort
> > +	operDeletePort
> > +)
> > +
> > +const (
> May be we should merge all these consts into one block? I am not sure if there is any golang specific convention here though.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/contiv/netplugin/pull/82/files#r31057914>.
>